includes/multi-currency/MultiCurrency.php

```diff
@@ -845,6 +845,45 @@ public function get_price( $price, string $type ): float {
 		return $this->get_adjusted_price( $converted_price, $apply_charm_pricing, $currency );
 	}
 
+	/**
+	 * Gets a raw converted amount based on the amount and currency codes passed.
+	 * This is a helper method for external conversions, if needed.
+	 *
+	 * @param float  $amount        The amount to be converted.
+	 * @param string $to_currency   The 3 letter currency code to convert the amount to.
+	 * @param string $from_currency The 3 letter currency code to convert the amount from.
+	 *
+	 * @return float The converted amount.
+	 *
+	 * @throws InvalidCurrencyException
+	 */
+	public function get_raw_conversion( float $amount, string $to_currency, string $from_currency = '' ): float {
+		$enabled_currencies = $this->get_enabled_currencies();
+
+		// If the from_currency is not set, use the store currency.
+		if ( '' === $from_currency ) {
+			$from_currency = $this->get_default_currency()->get_code();
+		}
+
+		// We throw an exception if either of the currencies are not enabled.
+		$to_currency   = strtoupper( $to_currency );
+		$from_currency = strtoupper( $from_currency );
+		foreach ( [ $to_currency, $from_currency ] as $code ) {
+			if ( ! isset( $enabled_currencies[ $code ] ) ) {
+				$message = 'Invalid currency passed to get_raw_conversion: ' . $code;
+				Logger::error( $message );
+				throw new InvalidCurrencyException( $message, 'wcpay_multi_currency_invalid_currency', 500 );
+			}
+		}
+
+		// Get the rates and convert the amount.
+		$to_currency_rate   = $enabled_currencies[ $to_currency ]->get_rate();
+		$from_currency_rate = $enabled_currencies[ $from_currency ]->get_rate();
```

Comment:
Can we add a `0` check here and throw if the from currency rate somehow returns falsy? It'll already throw a "division to zero" exception, but IMO it'll be good to own it.

Reply:
Added in https://github.com/Automattic/woocommerce-payments/pull/7202/commits/d9545cd1d884d1dae3b06634491ef2e29af14d56 . I had made the previous update, read the notes again, and realized I forgot to add it. Getting the testing to work was a pain, I kept trying to pass `BAN` as a currency into the mock arrays, then found that it had to be an actual currency code, so I changed it to `BAM` and it worked 🤦